ydb/core/blobstorage/pdisk/blobstorage_pdisk_impl.cpp

```diff
@@ -13,6 +13,8 @@
 
 #include <util/system/unaligned_mem.h>
 
+constexpr size_t MAX_REQS_PER_CYCLE = 200; // 200 requests take ~0.2ms in EnqueueAll function
```

Comment:
Maybe MAX_REQS_PER_CYCLE -> MAX_US_PER_CYCLE=200 and check timer below?

Reply:
Yes, it would be better if TInstant::Now() were very fast. But, unfortunately, it's very expensive to call Now() on every loop iteration, so we need to use workarounds